templates/mysql/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,42 @@
+version: '2'
+services:
+  mysql-lb:
+    image: rancher/lb-service-haproxy
+    links:
+      - mysql
+    ports:
+      - ${mysql_lb_port}:3306
```

Comment:
- Added https://github.com/rancher/community-catalog/pull/557/commits/69fe7ae6102ac988da88f2481b93f9d117337bee for the haproxy image version.
- The link is kept for semantics e.g. when looking at the link graph in the UI we can see that the two services have a networking relationship
![screen shot 2017-07-24 at 9 27 15 am](https://user-images.githubusercontent.com/131730/28503946-61d22af0-7052-11e7-8dce-cffd09b499bc.png)
- `${mysql_lb_port}:${mysql_lb_port}` - I believe this is incorrect; the variable, `mysql_lb_port` is only the public port used by the lb; the backend port will always be 3306 within the container where mysqld listens; if the user changes it in this case, it won't work